pkg/certificates/certificatesclient/client.go

```diff
@@ -0,0 +1,94 @@
+// Copyright (C) 2019 Storj Labs, Inc.
```

Comment:
Please can you add a `doc.go` file in this package with a brief description of the package?